packages/bundler-plugin-core/src/utils/providers/Bitrise.ts

```diff
@@ -11,41 +11,59 @@ export function detect(envs: ProviderEnvs): boolean {
   return Boolean(envs?.CI) && Boolean(envs?.BITRISE_IO);
 }
 
-function _getBuild(inputs: ProviderUtilInputs): ProviderServiceParams["build"] {
+function _getBuild(
+  inputs: ProviderUtilInputs,
+  output: Output,
+): ProviderServiceParams["build"] {
   const { args, envs } = inputs;
   if (args?.build && args.build !== "") {
     return args.build;
   }
-  return envs?.BITRISE_BUILD_NUMBER ?? null;
+  const build = envs?.BITRISE_BUILD_NUMBER ?? null;
+  debug(`Using build: ${build}`, { enabled: output.debug });
+  return build;
 }
 
 function _getBuildURL(
   inputs: ProviderUtilInputs,
+  output: Output,
 ): ProviderServiceParams["buildURL"] {
   const { envs } = inputs;
-  return envs?.BITRISE_BUILD_URL ?? null;
+  const buildURL = envs?.BITRISE_BUILD_URL ?? null;
+  debug(`Using buildURL: ${buildURL}`, { enabled: output.debug });
```

Comment:
I did a little checking around if any of these `buildURL`s contain any `?token=abcd` or sensitive info we shouldn't log - seems like we're good to go!